activation_logger.go

```diff
@@ -82,30 +82,26 @@ func NewActivation(config ActivationLoggerConfig) (Logger, error) {
 	return l, nil
 }
 
-func (l *activationLogger) Log(keyVals ...interface{}) error {
+func (l *activationLogger) Log(keyVals ...interface{}) {
 	activated, err := shouldActivate(l.activations, keyVals)
 	if err != nil {
-		return microerror.Mask(err)
+		panic(err)
```

Comment:
should we use `log` in this case or just panic?

Reply:
I think we should log. Panic will cause crashes.

Reply:
+1

I believe a log shouldn't panic while trying to log. Maybe we could improve the situation trying to log something with `fmt.print` to let the developer know that something is wrong.

Reply:
Thanks for the feedback, I'm using `log` just in case it's not working.